README.rst

```diff
@@ -1183,6 +1183,7 @@ the code: xfade~
     void xfade_tilde_setup(void) {
       xfade_tilde_class = class_new(gensym("xfade~"),
             (t_newmethod)xfade_tilde_new,
+            (t_method)xfade_tilde_free,
             0, sizeof(t_xfade_tilde),
```

Comment:
thanks for the PR, but I'm afraid I have to turn this down.

the original code already passes a destructor (well, the dummy destructor `0`), you cannot just *add* another argument.

instead replace the `0` with  `(t_method)xfade_tilde_free`


and while you are there, please submit spelling fixes in a *separate* commit.